sandbox/apparmor/profile.go

```diff
@@ -251,3 +303,86 @@ var SnapConfineDistroProfilePath = func() string {
 
 	return ""
 }
+
+var UserHomedirs = func() ([]string, error) {
```

Comment:
I don't think this should be exported, also not sure why the User part in the name?

Reply:
Ah right, that's correct